src/components/plots/helpers/heatmap/utils.js

```diff
@@ -0,0 +1,38 @@
+const hexToRgb = (hex) => {
```

Comment:
there is already a duplicate `hexToRgb` helper function in `src/utils/embeddingPlotHelperFunctions`. Since now you will be using this function in the heatmap too, better put it in a common place where both the embedding and the heatmap can access it

Reply:
I agree, good catch